src/core/server/config/config_service.mock.ts

```diff
@@ -22,7 +22,10 @@ import { ObjectToConfigAdapter } from './object_to_config_adapter';
 
 import { IConfigService } from './config_service';
 
-const createConfigServiceMock = () => {
+const createConfigServiceMock = ({
+  atPath = {},
+  getConfig$ = {},
+}: { atPath?: unknown; getConfig$?: Record<string, any> } = {}) => {
```

Comment:
with this approach `atPath` loses type safety. `const configService = configServiceMock.create({ atPath: 1 });`